integration/basic_test.go

```diff
@@ -132,6 +132,51 @@ func TestFindCommentQuery(t *testing.T) {
 	assert.Contains(t, databaseNames, name)
 }
 
+func TestUpdateCommentMethod(t *testing.T) {
```

Comment:
Would it make sense to combine this test and `TestUpdateCommentQuery` into one (with different test cases running as subtests)? Or is there a particular reason to keep those tests separately?

Also, none of the tests cover the case when `comment` is a part of the update doc, but such a code is implemented. If it's a possible case, let's write a test.

Reply:
> none of the tests cover the case when comment is a part of the update doc,

Please expand on that, because there are many "comment"s (`comment`, `$comment`) and many "update doc"s (`update`, `updates`, `u`) :)

Reply:
Ok, I take back my words :)

So, the full list of cases (as far as I understand).

For `db.collection.update(query, update, ..., comment)`
- `$comment` is a part of `query` (is it possible at all?)
- `comment` is a part of the list of `update` arguments

With that, there are 4 possible cases:
- none comments are set
- `$comment` as a part of `query` is set
- `comment` as a separate param is set
- both values are set

I believe we can write one test with four subtests (table test cases) to cover them all.

Reply:
Let's rephrase it in terms of https://www.mongodb.com/docs/manual/reference/command/update/, not in terms of mongosh method

Reply:
Fixed

Reply:
@rumyantseva then should we also implement something similar for the [find method](https://github.com/FerretDB/FerretDB/blob/main/integration/basic_test.go#L106-L133)?

Reply:
Nvm, found out the answer